common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1567,4 +1567,27 @@
 
         public static final String USER_SIGNED_OUT_FROM_SHARED_DEVICE = "user_signed_out_from_shared_device";
     }
+
+    public static final class SdkPlatformFields {
```

Comment:
Some javadoc for this public class would be good

Reply:
170586b